ceph-dev-new-trigger/config/definitions/ceph-dev-new-trigger.yml

```diff
@@ -172,6 +172,28 @@
                     DISTROS=centos8
                     FLAVOR=crimson
                     ARCHS=x86_64
+      # Build jaegertracing branch on needed env,  don't waste resources on the default one.
+      # Useful for testing specific builds failure
+      # default: focal
+      # crimson: centos8
+      - conditional-step:
+          condition-kind: regex-match
+          regex: .*jaeger-only.*
+          label: '${GIT_BRANCH}'
+          on-evaluation-failure: dont-run
+          steps:
+            - shell:
+                !include-raw:
+                - ../../../scripts/build_utils.sh
+                - ../../build/notify
+            - trigger-builds:
+                - project: 'ceph-dev-new'
+                  predefined-parameters: |
+                    BRANCH=${GIT_BRANCH}
+                    FORCE=True
+                    DISTROS=centos8 focal
+                    FLAVOR=crimson
```

Comment:
this is wrong.

Reply:
sure, we can keep them exclusive, but if a user builds ceph with both jaeger and crimson, they might hit build failures found crimson as more exhaustive testing set. But it's still in development, we can defer that :)

Reply:
> if branch will have `jaeger-only` in branch name, it shall only build
>  focal, crimson-centos8 x86_64

sorry, i assumed that you wanted to build jaeger only build using the same machinery i proposed at https://github.com/ceph/ceph-build/pull/1862#discussion_r665488883 . if you think otherwise. could you please elaborate in the commit message? as i don't understand how `jaeger-only` in the branch name  is connected to crimson flavor build.

Reply:
sure, I will add

Reply:
https://github.com/ceph/ceph-build/pull/1862#issue-682340359